From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect enum tcp_ca_event
Patch-mainline: never, kabi
References: kabi

In 4.4.152, commit 43707aa8c55fb165a1a56f590e0defb198ebdde9 (tcp: remove
DELAYED ACK events in DCTCP), upstream commit
a69258f7aa2623e0930212f09c586fd06674ad79 removed 2 entries from enum
tcp_ca_event. It made the kABI checker to complain.

Given the enum is not used for sizing any structure, just hide the
change from the kABI checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/net/tcp.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/include/net/tcp.h
+++ b/include/net/tcp.h
@@ -822,6 +822,10 @@ enum tcp_ca_event {
 	CA_EVENT_LOSS,		/* loss timeout */
 	CA_EVENT_ECN_NO_CE,	/* ECT set, but not CE marked */
 	CA_EVENT_ECN_IS_CE,	/* received CE marked IP packet */
+#ifdef __GENKSYMS__
+	CA_EVENT_DELAYED_ACK,
+	CA_EVENT_NON_DELAYED_ACK,
+#endif
 };
 
 /* Information about inbound ACK, passed to cong_ops->in_ack_event() */
